protobuf/xain_fl/grpc/coordinator.proto

```diff
@@ -48,29 +48,19 @@ message HeartbeatReply {
 message StartTrainingRequest {}
 
 message StartTrainingReply {
-  repeated numproto.protobuf.NDArray theta = 1;
+  repeated numproto.protobuf.NDArray weights = 1;
   int32 epochs = 2;
   int32 epoch_base = 3;
 }
 
 message EndTrainingRequest {
-  ThetaUpdate theta_update = 1;
-  map<string, HistoryValue> history = 2;
-  Metrics metrics = 3;
-
-  message ThetaUpdate {
-    repeated numproto.protobuf.NDArray theta_prime = 1;
-    int32 num_examples = 2;
-  }
-
-  /* protobuf doesn't like repeated as map values */
-  message HistoryValue {
-    repeated float values = 1;
-  }
+  repeated numproto.protobuf.NDArray weights = 1;
+  // number of training examples
```

Comment:
Do we really need this comment here?

Reply:
As long as it is stated in the sdk example and in the base class, I'm fine with removing the comment here.